build.gradle.kts

```diff
@@ -51,7 +51,7 @@ allprojects {
             dependsOn("cleanTest")
         }
         withType<JavaCompile> {
-            options.compilerArgs.addAll(arrayOf("-Xlint:all", "-Werror", "-Xlint:-classfile"))
+            options.compilerArgs.addAll(arrayOf("-Xlint:all", "-Werror", "-Xlint:-classfile", "-Xlint:-processing"))
```

Comment:
Adding Spring Boot `annotationProcessor` in Gradle config causes warnings that not all annotations get processed - which does not make much sense as we do not want to process all the annotations. That's why this warning gets turned off here.